agent/consul/autopilot_test.go

```diff
@@ -405,7 +405,7 @@ func TestAutopilot_BootstrapExpect(t *testing.T) {
 	conf := func(c *Config) {
 		c.Datacenter = dc
 		c.Bootstrap = false
-		c.BootstrapExpect = 3
+		c.BootstrapExpect = 4
```

Comment:
Presumably this helps because we can do a single leadership check and with that know that all 4 nodes have joined the cluster?

Reply:
Yep! Contemplating deleting the loop below this to have all servers check to make sure they have 4 members.  I'll test that now that I'm thinking about it.